includes/class-newspack-image-credits.php

```diff
@@ -369,7 +371,26 @@ public static function get_default_settings() {
 				'type'        => 'image',
 				'value'       => null,
 			],
+			[
+				'description' => __( 'Automatically populate image credits from EXIF or IPTC metadata when uploading new images.', 'newspack-image-credits' ),
+				'key'         => 'newspack_image_credits_auto_populate',
+				'label'       => __( 'Auto-populate image credits', 'newspack-image-credits' ),
+				'type'        => 'boolean',
+				'value'       => true,
```

Comment:
Looks like this should be false per this comment: https://github.com/Automattic/newspack-plugin/pull/2973#issuecomment-1978779652

Reply:
Updated in 91dd7bed